.github/workflows/ci.yml

```diff
@@ -74,6 +74,36 @@ jobs:
           NIX_TARGET: .#bpftrace-llvm18
           RUN_MEMLEAK_TEST: 1
           RUN_TESTS: 0
+        - NAME: LLVM 19 Release
+          CMAKE_BUILD_TYPE: Release
+          NIX_TARGET: .#bpftrace-llvm19
+          TOOLS_TEST_DISABLE: biosnoop.bt
+        - NAME: LLVM 19 Debug
+          CMAKE_BUILD_TYPE: Debug
+          NIX_TARGET: .#bpftrace-llvm19
+          TOOLS_TEST_DISABLE: biosnoop.bt
+        - NAME: LLVM 19 Clang Debug
+          CMAKE_BUILD_TYPE: Debug
+          NIX_TARGET: .#bpftrace-llvm19
+          CC: clang
+          CXX: clang++
+          TOOLS_TEST_DISABLE: biosnoop.bt
+        - NAME: AOT (LLVM 19 Debug)
+          CMAKE_BUILD_TYPE: Debug
+          NIX_TARGET: .#bpftrace-llvm19
+          RUN_TESTS: 0
+          RUN_AOT_TESTS: 1
+          AOT_SKIPLIST_FILE: .github/include/aot_skip.txt
+        - NAME: Memleak test (LLVM 19 Debug)
+          CMAKE_BUILD_TYPE: Debug
+          NIX_TARGET: .#bpftrace-llvm19
+          RUN_MEMLEAK_TEST: 1
+          RUN_TESTS: 0
+        - NAME: Memleak test (LLVM 19 Release)
+          CMAKE_BUILD_TYPE: Release
+          NIX_TARGET: .#bpftrace-llvm19
+          RUN_MEMLEAK_TEST: 1
+          RUN_TESTS: 0
```

Comment:
For the sake of limiting the amount of CI jobs, we tend to test all LLVM versions in the `Debug` and all the additional configs (`Release`, Clang, AOT, memleak) with just the latest LLVM.

Could you please do that? In other words, this PR should just add one new job `LLVM 19 Debug` and change LLVM 18->19 in all of the other jobs.